latch/types/metadata.py

```diff
@@ -651,3 +773,37 @@ def __post_init__(self):
 
 
 _snakemake_metadata: Optional[SnakemakeMetadata] = None
+
+
+@dataclass
+class NextflowMetadata(LatchMetadata):
+    name: Optional[str] = None
+    """
+    Name of the workflow
+    """
+    parameters: Dict[str, NextflowParameter] = field(default_factory=dict)
+    """
+    A dictionary mapping parameter names (strings) to `NextflowParameter` objects
+    """
+    runtime_resources: NextflowRuntimeResources = field(
+        default_factory=NextflowRuntimeResources
+    )
+    """
+    Resources (cpu/memory/storage) for Nextflow runtime task
+    """
+    output_dir: Optional[LatchDir] = None
```

Comment:
```suggestion
    log_dir: Optional[LatchDir] = None
```

also whats the behavior if this is `None`?